File: 10 - Python/Exercícios/05 - Matrizes/Exercício 03.py
#Exercício - Negativos Matriz

linha = int(input("Qual a quantidade de linhas da matriz? "))
coluna = int(input("Qual a quantidade de colunas da matriz? "))

mat = [[0 for x in range(coluna)] for x in range(linha)]

for i in range(linha):
    for j in range(coluna):
        mat[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("")
print("Valores negativos:")

for i in range(linha):
    for j in range(coluna):
        if mat[i][j] < 0:
            print(mat[i][j])

File: 07 - C++/Exercícios/05 - Matrizes/Exercício 03.cpp
//Exercício - Negativos Matriz

#include <iostream>

using namespace std;

int main(){

    int i, j, linha, coluna;

    cout << "Qual a quantidade de linhas da matriz? ";
    cin >> linha;
    cout << "Qual a quantidade de colunas da matriz? ";
    cin >> coluna;

    int mat[linha][coluna];

    for(i = 0; i < linha; i++){
        for(j = 0; j < coluna; j++){
            cout << "Elemento [" << i  << ", " << j << "]: ";
            cin >> mat[i][j];
        }
    }

    cout << endl << "Valores negativos:" << endl;

    for(i = 0; i < linha; i++){
        for(j = 0; j < coluna; j++){
            if(mat[i][j] < 0){
                cout << mat[i][j] << endl;
            }
        }
    }

    return 0;

}